NEO-TAT/tat_flutter
# chore: add Flutter Version Manager configuration

## Description

This PR adds a `.fvmrc` file and updates `.gitignore` and `.vscode/settings.json` to integrate with [Flutter Version Manager (FVM)](https://fvm.app/). This revisits and builds upon the work in #196.

## Implementation

- [x] Added a `.fvmrc` file to specify using Flutter 3.7.12 for this project.
- [x] Updated `.gitignore` to exclude the `.fvm` cache directory.
- [x] Configured `dart.flutterSdkPath` in VS Code to point to the FVM-managed Flutter SDK.

## Testing Instructions

To set up FVM and Flutter 3.7.12 (our last supported version):

1. Install the latest Dart SDK (e.g., `pacman -S dart`).
2. Run `dart pub global activate fvm` to install FVM.
3. Run `fvm use` in the project directory to install the Flutter version specified in `.fvmrc`.
4. Use `fvm flutter --version` to verify that Flutter is correctly installed.
